app/views/layouts/admin/base.haml

```diff
@@ -32,5 +32,5 @@
 
       .footer.text-right
         Version
-        = CURRENT_COMMIT_HASH
+        = link_to "#{CURRENT_COMMIT_HASH}", "#{CURRENT_COMMIT_REPO}/commits/#{CURRENT_COMMIT_HASH}", style: 'color:#000000;'
```

Comment:
Inline CSS should be avoided (it will be disabled entirely https://github.com/internetee/registry/issues/1271).

Reply:
I was not aware of that issue, fixed in: https://github.com/internetee/registry/pull/1345/commits/76de2d0f2e66052b106204ffea133d6ae8c9e47a